### PR TITLE
Adds ability to select a given sheet/tab by name

### PR DESCRIPTION
**Pull request by spookyuser**
This does a really simple check to see whether a user has specified a sheet name. If so, the script will then get that sheet instead of the sheet. However, if no sheet name is specified the script will behave exactly as before and fetch the first sheet by default.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,12 @@ parser.parse().then((items) => {
 parser.parse(spreadsheetId).then((items) => {
   // items should be [{"a":1,"b":2,"c":3},{"a":4,"b":5,"c":6},{"a":7,"b":8,"c":9}]
 })
+
+
+// 4. You can also pass  the name of specific sheet/tab if you want to get that instead of the first one.
+parser.parse(spreadsheetId, "Sheet2").then((items) => {
```
Comment (fureweb-com) [inline README.md:60]: Please replace double quote to single quote :)

File: src/index.js [hunk 2]
```diff
@@ -2,18 +2,25 @@ const isBrowser = typeof require === 'undefined'
 const fetch = isBrowser ? window.fetch : require('node-fetch')
 
 class PublicGoogleSheetsParser {
-  constructor (spreadsheetId) {
+  constructor (spreadsheetId, tabName) {
```
Comment (fureweb-com) [inline src/index.js:5]: Could you rename from tabName to sheetName?

according to below:
https://developers.google.com/sheets/api/guides/concepts

They don't use 'tab'.

File: src/index.js [hunk 3]
```diff
@@ -2,18 +2,25 @@ const isBrowser = typeof require === 'undefined'
...
     this.id = spreadsheetId
+    this.tab = tabName
```
Comment (fureweb-com) [inline src/index.js:7]: I think it would be better to use sheetName rather than tab for this variable name.

File: test/index.js [hunk 4]
```diff
@@ -62,3 +62,20 @@ test('parse method should return array even spreadsheetId is invalid', async (t)
...
+  const spreadsheetId = '10WDbAPAY7Xl5DT36VuMheTPTTpqx9x0C5sDCnh4BGps'
+  const result = await parser.parse(spreadsheetId)
+  const expected = [{ a: 1, b: 2, c: 3 }, { a: 4, b: 5, c: 6 }, { a: 7, b: 8, c: 9 }]
+  t.deepEqual(result, expected)
+  t.end()
+})
+
+test('should get 2nd tab if specified', async (t) => {
+  const spreadsheetId = '1wdU-Cf7Kn5ZvhcvRePOoL10urdTA6TWGndBSKDAiYbQ'
+  const tabName = 'Sheet2'
```
Comment (fureweb-com) [inline test/index.js:76]: Please change from tabName to sheetName for consistency!

File: src/index.js [hunk 5]
```diff
@@ -49,8 +56,9 @@ class PublicGoogleSheetsParser {
     return rows
   }
 
-  async parse (spreadsheetId) {
+  async parse (spreadsheetId, tabName) {
     if (spreadsheetId) this.id = spreadsheetId
+    if (tabName) this.tab = tabName
```
Comment (fureweb-com) [inline src/index.js:61]: Please check here to match the changes above I mentioned!

**Comment by fureweb-com** [comment]
Thank you for your interest and suggesting a feature like this!

There are parts that I would like to be modified according to my personal preferences, so I added a comment on this part.

If you can correct the part, I want to merge it immediately after confirmation.

Thanks again for the feature suggestion :)

**Comment by spookyuser** [comment]
Of course, will fix it all right now 💯

File: src/index.js (hunk 2 above)
Comment (spookyuser) [reply src/index.js:5]: Good point lol, I think I got the word tab from the stackoverflow question 😆

**Review by fureweb-com — APPROVED** [review]
Thanks! 👍

**Pull request merged**